src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -1028,23 +1024,18 @@ private Map<String, BigDecimal> updateCandidateNamesInTally(RoundTally roundSumm
 
   private Map<String, BigDecimal> getInactiveJsonMap(RoundTally roundTally) {
     Map<String, BigDecimal> inactiveMap = new HashMap<>();
-    List<Pair<String, StatusForRound>> statusesToPrint = new ArrayList<>();
-    statusesToPrint.add(new Pair<>("overvotes",
-            StatusForRound.INVALIDATED_BY_OVERVOTE));
-    statusesToPrint.add(new Pair<>("skippedRankings",
-            StatusForRound.INVALIDATED_BY_SKIPPED_RANKING));
-    statusesToPrint.add(new Pair<>("exhaustedChoices",
-            StatusForRound.EXHAUSTED_CHOICE));
-    statusesToPrint.add(new Pair<>("repeatedRankings",
-            StatusForRound.INVALIDATED_BY_REPEATED_RANKING));
+    List<StatusForRound> statusesToPrint = new ArrayList<>();
+    statusesToPrint.add(StatusForRound.INVALIDATED_BY_OVERVOTE);
+    statusesToPrint.add(StatusForRound.INVALIDATED_BY_SKIPPED_RANKING);
+    statusesToPrint.add(StatusForRound.EXHAUSTED_CHOICE);
+    statusesToPrint.add(StatusForRound.INVALIDATED_BY_REPEATED_RANKING);
     if (config.usesSurpluses()
         && roundTally.getRoundNumber() == numRounds) {
-      statusesToPrint.add(new Pair<>("finalRoundSurplus",
-              StatusForRound.FINAL_ROUND_SURPLUS));
+      statusesToPrint.add(StatusForRound.FINAL_ROUND_SURPLUS);
     }
-    for (Pair<String, StatusForRound> statusToPrint : statusesToPrint) {
+    for (StatusForRound statusToPrint : statusesToPrint) {
       inactiveMap.put(
-          statusToPrint.getKey(), roundTally.getBallotStatusTally(statusToPrint.getValue()));
+          statusToPrint.getCamelCaseKey(), roundTally.getBallotStatusTally(statusToPrint));
```

Comment:
I see that you didn't love the stream suggestion! Since you're using a modifiable map here, you could use the static final immutable list described above like this:
```
private Map<String, BigDecimal> getInactiveJsonMap(RoundTally roundTally) {
    Map<String, BigDecimal> result = new HashMap<>();
    for (StatusForRound statusToPrint : STATUSES_TO_PRINT) {
        result.put(statusToPrint.getCamelCaseKey(), roundTally.getBallotStatusTally(statusToPrint));
    }

    if (config.usesSurpluses() && roundTally.getRoundNumber() == numRounds) {
        result.put(FINAL_ROUND_SURPLUS.getCamelCaseKey(), roundTally.getBallotStatusTally(FINAL_ROUND_SURPLUS));
    }

    return result;
}
```

Again, not asking for changes in this PR! Just pointing out what I see :) If it's useful to you, great! Overall, I think the changes that have already been made are a step in the right direction.